### DISCUSSION TITLE
macOS - "Unable to find GDB in default search path and /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lldb."

### DISCUSSION THREAD

**Discussion by sean-parent** [microsoft/vscode-cmake-tools#3701] [Q&A] [answered]
I receive this error message when selecting `CMake: Debug`.

Unable to find GDB in default search path and /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lldb.

Xcode 15.3 doesn't place lldb at that location. Is there a way to configure cmake-tools with the location of lldb?

I couldn't find any examples of using `cmake.debugConfig,` but setting it to this got me going:
```
    "cmake.debugConfig": {
        "type": "cppdbg",
        "MIMode": "lldb"
    }
```
Is there a way to add `/usr/bin/lldb` to the default search path? The documentation for `debugConfig` specifies a configuration but not in what form, and I couldn't find any online example. I'm still not clear if I'm doing it "correctly."

<img width="464" alt="image" src="https://github.com/microsoft/vscode-cmake-tools/assets/2279724/78d2e2c7-0741-4c53-843e-8c919d8a9e66">

**Comment by gcampbell-msft** [comment, marked as answer]
#514 Seems to track this as well. I'll close this in favor of tracking it there. If you have additional context from this discussion that you'd like to add to that issue, please feel free and vote on the issue. Thanks!
